shared/oae/api/oae.api.admin.js

```diff
@@ -23,14 +23,16 @@ define(['exports', 'jquery', 'underscore'], function(exports, $, _) {
     /**
      * Create a new tenant
      *
-     * @param  {String}                alias                               The alias of the tenant to create
-     * @param  {String}                displayName                         The display name of the tenant to create
-     * @param  {String}                host                                The host name of the tenant to create
-     * @param  {Function}              [callback]                          Standard callback method
-     * @param  {Object}                [callback.err]                      Error object containing error code and error message
-     * @param  {Tenant}                [callback.response]                 A Tenant object representing the created tenant
+     * @param  {String}         alias                               The alias of the tenant to create
+     * @param  {String}         displayName                         The display name of the tenant to create
+     * @param  {String}         host                                The host name of the tenant to create
+     * @param  {Object}         [opts]                              Optional arguments
+     * @param  {String}         [opts.emailDomain]                  The email domain to assign to the tenant
+     * @param  {Function}       [callback]                          Standard callback method
+     * @param  {Object}         [callback.err]                      Error object containing error code and error message
+     * @param  {Tenant}         [callback.response]                 A Tenant object representing the created tenant
```

Comment:
I think I changed this already in the tenant paging PR, so I'll just wait for that to be merged together.